state/tx_filter_test.go

```diff
@@ -16,7 +16,7 @@ import (
 
 func TestTxFilter(t *testing.T) {
 	genDoc := randomGenesisDoc()
-	genDoc.ConsensusParams.Block.MaxBytes = 3000
+	genDoc.ConsensusParams.Block.MaxBytes = 3080
```

Comment:
Reasoning for this change?

Reply:
There is a difference of 48 in the max evidence size and 32 in the max vote size so 80 total for 1 evidence and 1 validator.